packages/create-plugin/templates/panel/provisioning/dashboards/dashboard.json

```diff
@@ -0,0 +1,130 @@
+{
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": {
+            "type": "grafana",
+            "uid": "-- Grafana --"
+          },
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
+          "name": "Annotations & Alerts",
+          "type": "dashboard"
+        }
+      ]
+    },
+    "editable": true,
+    "fiscalYearStartMonth": 0,
+    "graphTooltip": 0,
+    "links": [],
+    "liveNow": false,
+    "panels": [
+      {
+        "datasource": {
+          "type": "dashboard",
+          "uid": "-- Dashboard --"
```

Comment:
Scaffoling a panel plugin gives me the following error when I navigate to the dashboard:

![image](https://github.com/grafana/plugin-tools/assets/73201/39c33ebd-0213-495f-9cde-ba83243dbdd7)

I believe it's because this datasource expects other panels to use as a reference and cannot find any. I think a simple fix would be to use the default random walk which can be done with:

```suggestion
        "type": "grafana",
        "uid": "grafana"
```